README.md

```diff
@@ -4,14 +4,18 @@
 
 ---
 
-[![Build Status](https://jenkins.blueoil.org/job/blueoil_main/badge/icon)](https://jenkins.blueoil.org/job/blueoil_main/)
-[![Build Status](https://jenkins.blueoil.org/job/blueoil_lmnet/badge/icon)](https://jenkins.blueoil.org/job/blueoil_lmnet/)
-[![CircleCI](https://circleci.com/gh/blue-oil/blueoil.svg?style=svg)](https://circleci.com/gh/blue-oil/blueoil)
-
 Blueoil provides two features.
 * Training a neural network model
 * Converting a trained model to an executable binary (or library), which utilize FPGAs for acceleration.
 
+| Type | Status |
```

Comment:
<img width="366" alt="スクリーンショット 2019-07-12 13 19 33" src="https://user-images.githubusercontent.com/478824/61102184-d0239f80-a4a7-11e9-8e3f-72462910f0fd.png">

Do you plan to remove jenkins icon before this PR merge?
Both Jenkins and Buildkite icon looks same. It may occur some confusion.

Reply:
we will operate jenkins and buildkite for 2 weeks. So we will have 2 icons during the term and after that, we will remove jenkins one.

Reply:
@kchygoe I see.
I think it's good to add "Status[Jenins]" and "Status[Buildkite]" .

Reply:
@iizukak Do you mean such as follows?
![image](https://user-images.githubusercontent.com/12394960/61105600-509ccd00-a4b5-11e9-9b39-481ffe66cc7b.png)

Reply:
I updated status tables as follows.
Buildkite -> New CI
![image](https://user-images.githubusercontent.com/12394960/61112283-741c4380-a4c6-11e9-91bc-140d15a30a7a.png)